src/App.css

```diff
@@ -1,10 +1,11 @@
+/* stylelint-disable no-descending-specificity */
```

Comment:
- [ ] [OPTIONAL] Kindly consider fixing the linters error if there is any, but don't skip them, they are in place to make your code better